src/NuGet.Core/NuGet.Packaging.Core.Types/PackagingCoreConstants.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace NuGet.Packaging.Core
+{
+    public static class PackagingCoreConstants
+    {
+        public const string NupkgExtension = ".nupkg";
```

Comment:
static readonly like the strings below?

aside - Did we ever have them at different casing (please say no :) )